.github/semantic.yaml

```diff
@@ -0,0 +1,59 @@
+###############################################################################
+# This file configures "Semantic Pull Requests", which is documented here:
+# https://github.com/zeke/semantic-pull-requests
+###############################################################################
+
+# Scopes are optionally supplied after a 'type'. For example, in
+#
+# feat(cdr): autostart ui
+#
+# '(cdr)' is the scope. Scopes are used to signify where the change occurred.
+scopes: []
```

Comment:
need some help filling in the acceptable scopes here, do we have a list of scopes we use?

Reply:
I think @code-asher and @jsjoeio should figure this one out. From my take, could be:

- docs
- vender

as optional scopes when those components are modified.

Reply:
`docs` and `vendor` sound reasonable to me I have also seen @jsjoeio using `tests` and `deps` I think.

Reply:
sgtm!

Reply:
- docs
- vendor
- tests
- deps
- cs (for code-server)
- cli

Reply:
tests seems unnecessary for scopes since there's already a `test` type? also, is there a meaningful distinction between deps and vendor?

Reply:
Good to know. `test` is probably fine then. 

> also, is there a meaningful distinction between deps and vendor?

yes! `deps` relate to dependencies in code-server while `vendor` relates to changes for `vscode` (which lives under `vendor/package.json` - each  time a change is made to coder/vscode, we have to update the commit it points to)

Reply:
@jsjoeio cool! I think everything should be good now, can you take another look?